tools/run_mview_sperson_estimator.py

```diff
@@ -0,0 +1,217 @@
+        os.makedirs(keypoints2d_dir, exist_ok=True)
+        for keypoints2d, img_list_path in zip(keypoints2d_list,
+                                              image_list_paths):
+            view_idx = int(
+                os.path.basename(img_list_path).rsplit('.',
+                                                       1)[0].split('_')[-1])
+            keypoints2d_path = os.path.join(
+                keypoints2d_dir, f'keypoints2d_view{view_idx:02d}.npz')
+            if keypoints2d is not None:
+                keypoints2d.dump(keypoints2d_path)
+            else:
+                logger.warning(
+                    f'No keypoints2d has been detected in view{view_idx:02d}.')
+        keypoints3d_path = os.path.join(scene_path, 'keypoints3d_pred.npz')
+        keypoints3d.dump(keypoints3d_path)
+        if isinstance(smpl_data, SMPLXData):
+            smpl_type = 'smplx'
+        else:
+            smpl_type = 'smpl'
+        smpl_path = os.path.join(scene_path, f'{smpl_type}_data.npz')
+        smpl_data.dump(smpl_path)
+        if args.visualize:
+            vis_dir = os.path.join(args.meta_path, 'visualize')
+            scene_name = os.path.basename(scene_path)
+            scene_vis_dir = os.path.join(vis_dir, scene_name)
+            os.makedirs(scene_vis_dir, exist_ok=True)
+            if keypoints3d is not None:
+                # visualize keypoints in a 3D scene
+                logger.info('Visualizing keypoints3d.')
+                visualize_keypoints3d(
+                    keypoints=keypoints3d,
+                    output_path=os.path.join(scene_vis_dir,
+                                             'keypoints3d_pred.mp4'),
+                    overwrite=True,
+                    plot_axis=True,
+                    plot_points=True,
+                    plot_lines=True,
+                    disable_tqdm=False,
+                    logger=logger)
+                for idx, img_list_path in enumerate(image_list_paths):
+                    view_idx = int(
+                        os.path.basename(img_list_path).rsplit(
+                            '.', 1)[0].split('_')[-1])
+                    logger.info(
+                        f'Visualizing keypoints3d from view {view_idx:02d}.')
+                    # visualize projected keypoints3d
+                    visualize_keypoints3d_projected(
+                        keypoints=keypoints3d,
+                        camera=cam_param_list[idx],
+                        output_path=os.path.join(
+                            scene_vis_dir,
+                            f'keypoints3d_pred_projected_{view_idx:02d}.mp4'),
+                        overwrite=True,
+                        plot_points=True,
+                        plot_lines=True,
+                        background_img_list=mview_img_list[idx],
+                        disable_tqdm=False,
+                        logger=logger)
+                    break  # TODO: remove this break
```

Comment:
```suggestion
```

Reply:
Removed in [83ac66b](https://github.com/openxrlab/xrmocap/pull/123/commits/83ac66b600c1681e720382df169657ac87ed3b88).